src/version.txt

```diff
@@ -1 +1 @@
-1.1.0
+1.1.1
```

Comment:
I *think* this constitutes a minor version bump, eg: 1.2.0, since we're adding a team, not just doing minor corrections, but I need to read what past Jake put in the README again.

Reply:
I'll bump, push, and squash